### PR TITLE
add all chrysler vehicles to common.js

### PR DESCRIPTION
**Pull request by hasalassh**
added all chrysler vehicles to common.js #21

### CODE REVIEW

File: common.js [hunk 1]
```diff
@@ -4043,6 +4043,524 @@ const allSoloCars = {
...
+    'Conquest & Starion (non-turbo)': {
+      'all': ['dsp']
+    },
+    'Daytona Turbo': {
+      'all': ['dsp']
+    },
+    'Daytona (V6)': {
+      'all': ['dsp']
+    },
+    'GLH-S & GLH Turbo': {
```
Comment (Bjorn248) [inline common.js:4591]: This is not a Chrysler, it is a Dodge

File: common.js [hunk 2]
```diff
@@ -4043,6 +4043,524 @@ const allSoloCars = {
...
+      'all': ['dsp']
+    },
+    'Laser Turbo (NOC) & K-car Turbo': {
+      'all': ['dsp']
+    },
+    'Shadow (4-cyl Turbo & V6)': {
+      'all': ['dsp']
+    },
+    'Shelby Charger Turbo': {
```
Comment (Bjorn248) [inline common.js:4600]: This is a Dodge...but should probably be listed under Shelby.

File: common.js [hunk 3]
```diff
@@ -4043,6 +4043,524 @@ const allSoloCars = {
...
+      'all': ['dsp']
+    },
+    'Barracuda': {
+      '1965': ['cp', 'esp'],
+      '1966': ['cp', 'esp'],
+      '1967': ['cp', 'esp'],
+      '1968': ['cp', 'esp'],
+      '1969': ['cp', 'esp']
+    },
+    'Dart, Duster': {
```
Comment (Bjorn248) [inline common.js:4619]: This is a Dodge and Plymouth

File: common.js [hunk 4]
```diff
@@ -4043,6 +4043,524 @@ const allSoloCars = {
...
+      '1976': ['esp']
+    },
+    'Barracuda & Challenger (E-body)': {
+      '1970': ['esp'],
+      '1971': ['esp'],
+      '1972': ['esp'],
+      '1973': ['esp'],
+      '1974': ['esp']
+    },
+    'Challenger': {
```
Comment (Bjorn248) [inline common.js:4645]: This is a Dodge

File: common.js [hunk 5]
```diff
@@ -4043,6 +4043,524 @@ const allSoloCars = {
...
+      '2008': ['esp'],
+      '2009': ['esp'],
+      '2010': ['esp'],
+      '2011': ['esp'],
+      '2012': ['esp'],
+      '2013': ['esp']
+    },
+    'Challenger (6-cyl & V8, NOC)': {
```
Comment (Bjorn248) [inline common.js:4653]: This is a Dodge

File: common.js [hunk 6]
```diff
@@ -4043,6 +4043,524 @@ const allSoloCars = {
...
+      'all': ['esp']
+    },
+    'Charger': {
```
Comment (Bjorn248) [inline common.js:4656]: This is a Dodge

File: common.js [hunk 7]
```diff
@@ -4043,6 +4043,524 @@ const allSoloCars = {
...
+      '2012': ['esp'],
+      '2013': ['esp']
+    },
+    'Laser (FWD)': {
+      'all': ['esp']
+    },
+    'Stealth (non-turbo)': {
+      'all': ['esp']
+    },
+    'Dakota': {
```
Comment (Bjorn248) [inline common.js:4672]: This is a Dodge

File: common.js [hunk 8]
```diff
@@ -4432,6 +4431,524 @@ const allSoloCars = {
...
+      '2009': ['esp'],
+      '2010': ['esp'],
+      '2011': ['esp'],
+      '2012': ['esp'],
+      '2013': ['esp']
+    },
+    'Laser (FWD)': {
+      'all': ['esp']
+    },
+    'Stealth (non-turbo)': {
```
Comment (Bjorn248) [inline common.js:4669]: This is a Dodge

File: common.js [hunk 9]
```diff
@@ -4432,6 +4431,524 @@ const allSoloCars = {
...
+      '1997': ['esp'],
+      '1998': ['esp'],
+      '1999': ['esp'],
+      '2000': ['esp'],
+      '2001': ['esp'],
+      '2002': ['esp'],
+      '2003': ['esp'],
+      '2004': ['esp']
+    },
+    'Acclaim (4-cyl non-turbo)': {
```
Comment (Bjorn248) [inline common.js:4682]: This is a Plymouth

File: common.js [hunk 10]
```diff
@@ -4432,6 +4431,524 @@ const allSoloCars = {
...
+      'all': ['fsp']
+    },
+    'Arrow 1600, 2000, & 2600': {
+      'all': ['fsp']
+    },
+    'Champ (non-turbo, all)': {
```
Comment (Bjorn248) [inline common.js:4688]: This is a Plymouth

File: common.js [hunk 11]
```diff
@@ -4432,6 +4431,524 @@ const allSoloCars = {
...
+      'all': ['fsp']
+    },
+    'Colt (non-turbo, FWD)': {
```
Comment (Bjorn248) [inline common.js:4691]: This is a Dodge

File: common.js [hunk 12]
```diff
@@ -4432,6 +4431,524 @@ const allSoloCars = {
...
+      'all': ['fsp']
+    },
+    'Colt (8v non-turbo)': {
```
Comment (Bjorn248) [inline common.js:4694]: This is a Dodge

File: common.js [hunk 13]
```diff
@@ -4432,6 +4431,524 @@ const allSoloCars = {
...
+      'all': ['fsp']
+    },
+    'Colt (1600 & 2000, RWD)': {
```
Comment (Bjorn248) [inline common.js:4697]: This is a Dodge

File: common.js [hunk 14]
```diff
@@ -4432,6 +4431,524 @@ const allSoloCars = {
...
+      'all': ['fsp']
+    },
+    'Daytona (non-turbo)': {
```
Comment (Bjorn248) [inline common.js:4700]: This is a Dodge

File: common.js [hunk 15]
```diff
@@ -4432,6 +4431,524 @@ const allSoloCars = {
...
+      '1992': ['fsp'],
+      '1993': ['fsp'],
+      '1994': ['fsp'],
+      '1995': ['fsp'],
+      '1996': ['fsp'],
+      '1997': ['fsp'],
+      '1998': ['fsp'],
+      '1999': ['fsp']
+    },
+    'Omni, 024, & Charger': {
```
Comment (Bjorn248) [inline common.js:4719]: This is a Dodge

File: common.js [hunk 16]
```diff
@@ -4432,6 +4431,524 @@ const allSoloCars = {
...
+      'all': ['fsp']
+    },
+    'Rampage (2.2L)': {
+      'all': ['fsp']
+    },
+    'Sapporo (1600, 2000, & 2600)': {
```
Comment (Bjorn248) [inline common.js:4725]: This is a Plymouth

File: common.js [hunk 17]
```diff
@@ -4432,6 +4431,524 @@ const allSoloCars = {
...
+      'all': ['fsp']
+    },
+    'Shelby (2.2L non-turbo)': {
+      '1983': ['fsp'],
+      '1984': ['fsp']
+    },
+    'Spirit (4-cyl non-turbo)': {
```
Comment (Bjorn248) [inline common.js:4732]: This is a Dodge

File: common.js [hunk 18]
```diff
@@ -4432,6 +4431,524 @@ const allSoloCars = {
...
+      'all': ['fsp']
+    },
+    '300 (all) ': {
```
Comment (Bjorn248) [inline common.js:4735]: This should be merged with the existing 300 models you have above. Remove this model and simply add `cp` to every year of both the V6 and V8 models you already have.

File: common.js [hunk 19]
```diff
@@ -4432,6 +4431,524 @@ const allSoloCars = {
...
+      '2010': ['cp'],
+      '2011': ['cp'],
+      '2012': ['cp'],
+      '2013': ['cp'],
+      '2014': ['cp'],
+      '2015': ['cp'],
+      '2016': ['cp'],
+      '2017': ['cp']
+    },
+    'A-body Valiant, Dart, Duster, Demon, etc.,': {
```
Comment (Bjorn248) [inline common.js:4749]: Should just be Valiant or A-body Valiant. The rest are Dodge or Plymouth models

File: common.js [hunk 20]
```diff
@@ -4432,6 +4431,524 @@ const allSoloCars = {
...
+      '1963': ['cp'],
+      '1964': ['cp'],
+      '1965': ['cp'],
+      '1966': ['cp'],
+      '1967': ['cp']
+    },
+    'Dakota 2WD': {
```
Comment (Bjorn248) [inline common.js:4756]: This is a Dodge

File: common.js [hunk 21]
```diff
@@ -4432,6 +4431,524 @@ const allSoloCars = {
...
+      '1997': ['cp'],
+      '1998': ['cp'],
+      '1999': ['cp'],
+      '2000': ['cp'],
+      '2001': ['cp'],
+      '2002': ['cp'],
+      '2003': ['cp'],
+      '2004': ['cp']
+    },
+    'Challenger (non-supercharged)': {
```
Comment (Bjorn248) [inline common.js:4776]: This is a Dodge

File: common.js [hunk 22]
```diff
@@ -4432,6 +4431,524 @@ const allSoloCars = {
...
+      '2010': ['cp'],
+      '2011': ['cp'],
+      '2012': ['cp'],
+      '2013': ['cp'],
+      '2014': ['cp'],
+      '2015': ['cp'],
+      '2016': ['cp'],
+      '2017': ['cp']
+    },
+    'Charger (non-supercharged)': {
```
Comment (Bjorn248) [inline common.js:4788]: This is a Dodge

File: common.js [hunk 23]
```diff
@@ -4432,6 +4431,524 @@ const allSoloCars = {
...
+      '2010': ['cp'],
+      '2011': ['cp'],
+      '2012': ['cp'],
+      '2013': ['cp'],
+      '2014': ['cp'],
+      '2015': ['cp'],
+      '2016': ['cp'],
+      '2017': ['cp']
+    },
+    'E-body Barracuda & Challenger': {
```
Comment (Bjorn248) [inline common.js:4802]: This is a Plymouth and Dodge

File: common.js [hunk 24]
```diff
@@ -4432,6 +4431,524 @@ const allSoloCars = {
...
+      '1970': ['cp'],
+      '1971': ['cp'],
+      '1972': ['cp'],
+      '1973': ['cp'],
+      '1974': ['cp']
+    },
+    'Colt & Champ (non-turbo)': {
```
Comment (Bjorn248) [inline common.js:4809]: These are Dodge and Plymouth

File: common.js [hunk 25]
```diff
@@ -4432,6 +4431,524 @@ const allSoloCars = {
...
+      '1979': ['ep'],
+      '1980': ['ep'],
+      '1981': ['ep'],
+      '1982': ['ep'],
+      '1983': ['ep']
+    },
+    'Colt & Mirage (non-turbo)': {
```
Comment (Bjorn248) [inline common.js:4816]: Not a Chrysler

File: common.js [hunk 26]
```diff
@@ -4432,6 +4431,524 @@ const allSoloCars = {
...
+      '1985': ['ep'],
+      '1986': ['ep'],
+      '1987': ['ep'],
+      '1988': ['ep'],
+      '1993': ['ep'],
+      '1994': ['ep'],
+      '1995': ['ep'],
+      '1996': ['ep']
+    },
+    'Colt, Mirage, & Summit (non-turbo) ': {
```
Comment (Bjorn248) [inline common.js:4827]: Not a Chrysler

File: common.js [hunk 27]
```diff
@@ -4432,6 +4431,524 @@ const allSoloCars = {
...
+      '1989': ['ep'],
+      '1990': ['ep'],
+      '1991': ['ep'],
+      '1992': ['ep']
+    },
+    'Daytona & Laser (2.2 L non-turbo)': {
```
Comment (Bjorn248) [inline common.js:4833]: The Daytona is not a Chrysler

File: common.js [hunk 28]
```diff
@@ -4432,6 +4431,524 @@ const allSoloCars = {
...
+      '1984': ['ep'],
+      '1985': ['ep'],
+      '1986': ['ep'],
+      '1987': ['ep'],
+      '1988': ['ep'],
+      '1989': ['ep'],
+      '1990': ['ep']
+    },
+    'Eclipse, Laser, & Talon (16v & 8v non-turbo, FWD)': {
```
Comment (Bjorn248) [inline common.js:4842]: Only the Laser is a Chrysler

File: common.js [hunk 29]
```diff
@@ -4432,6 +4431,524 @@ const allSoloCars = {
...
+      '1987': ['fp'],
+      '1988': ['fp'],
+      '1989': ['fp'],
+      '1990': ['fp'],
+      '1991': ['fp'],
+      '1992': ['fp'],
+      '1993': ['fp'],
+      '1994': ['fp']
+    },
+    'SRT-4 (Neon chassis)': {
```
Comment (Bjorn248) [inline common.js:4936]: This is a Dodge

File: common.js [hunk 30]
```diff
@@ -4432,6 +4431,524 @@ const allSoloCars = {
...
+    },
+    'Talon & Laser (Turbo, FWD & AWD)': {
+      '1989': ['fp'],
+      '1990': ['fp'],
+      '1991': ['fp'],
+      '1992': ['fp'],
+      '1993': ['fp'],
+      '1994': ['fp']
+    },
+    'Conquest & Starion Turbo': {
```
Comment (Bjorn248) [inline common.js:4949]: Starion is a Mistubishi

**Review by Bjorn248 — CHANGES_REQUESTED** [review]
Lots of models here that are not Chryslers.

File: common.js [hunk 31]
```diff
@@ -4043,6 +4043,524 @@ const allSoloCars = {
...
+    },
+    'Neon (NOC)': {
+      '2000': ['sts'],
+      '2001': ['sts'],
+      '2002': ['sts'],
+      '2003': ['sts'],
+      '2004': ['sts'],
+      '2005': ['sts']
+    },
+    'Neon R/T & ACR': {
```
[the remark this answers is not in the document]
Comment (hasalassh) [reply common.js]: fixed

**Comment by hasalassh** [comment]
@Bjorn248  I have done the modifications as per review comments

File: common.js [hunk 32]
```diff
@@ -4432,6 +4431,361 @@ const allSoloCars = {
       'Cobra 289': {
           'all': ['asp', 'bs', 'xp']
       }
+  },
+  'Chrysler': {
+    'Prowler': {
```
Comment (Bjorn248) [inline common.js:4436]: This is not a Chrysler

File: common.js [hunk 33]
```diff
@@ -4432,6 +4431,361 @@ const allSoloCars = {
...
+      '2004': ['sts'],
+      '2005': ['sts']
+    },
+    'Crossfire SRT6': {
+      'all': ['bsp']
+    },
+    'Acclaim (V6 & Turbo)': {
+      'all': ['dsp']
+    },
+    'Charger GLH-S': {
```
Comment (Bjorn248) [inline common.js:4582]: This is not a Chrysler

File: common.js [hunk 34]
```diff
@@ -4432,6 +4431,361 @@ const allSoloCars = {
...
+      'all': ['dsp']
+    },
+    'Conquest & Starion (non-turbo)': {
```
Comment (Bjorn248) [inline common.js:4585]: Starion is not a Chrysler

File: common.js [hunk 35]
```diff
@@ -4432,6 +4431,361 @@ const allSoloCars = {
...
+    'Spirit (4-cyl Turbo & V6)': {
+      'all': ['dsp']
+    },
+    'SRT-4': {
+      'all': ['dsp']
+    },
+    'Sundance Turbo': {
+      'all': ['dsp']
+    },
+    'Barracuda': {
```
Comment (Bjorn248) [inline common.js:4609]: This is not a Chrysler

File: common.js [hunk 36]
```diff
@@ -4432,6 +4431,361 @@ const allSoloCars = {
...
+      '1969': ['esp'],
+      '1970': ['esp'],
+      '1971': ['esp'],
+      '1972': ['esp'],
+      '1973': ['esp'],
+      '1974': ['esp'],
+      '1975': ['esp'],
+      '1976': ['esp']
+    },
+    'Barracuda & Challenger (E-body)': {
```
Comment (Bjorn248) [inline common.js:4632]: This is not a Chrysler

File: common.js (hunk 22 above)
Comment (Bjorn248) [reply common.js:4675]: Not a Chrysler

File: common.js [hunk 37]
```diff
@@ -4432,6 +4431,361 @@ const allSoloCars = {
...
+      '1987': ['ep'],
+      '1988': ['ep'],
+      '1989': ['ep'],
+      '1990': ['ep'],
+      '1991': ['ep'],
+      '1992': ['ep'],
+      '1993': ['ep'],
+      '1994': ['ep']
+    },
+    'Shelby Charger': {
```
Comment (Bjorn248) [inline common.js:4737]: This is not a Chrysler

File: common.js [hunk 38]
```diff
@@ -4432,6 +4431,361 @@ const allSoloCars = {
...
+      '1991': ['ep'],
+      '1992': ['ep'],
+      '1993': ['ep'],
+      '1994': ['ep'],
+      '1995': ['ep']
+    },
+    'Sedans (N/A, FWD, NOC)': {
+      'all': ['ep']
+    },
+    'Colt Turbo': {
```
Comment (Bjorn248) [inline common.js:4757]: This is not a Chrysler

File: common.js [hunk 39]
```diff
@@ -4432,6 +4431,361 @@ const allSoloCars = {
...
+    },
+    'Daytona & Laser (Turbo)': {
+      '1984': ['fp'],
+      '1985': ['fp'],
+      '1986': ['fp'],
+      '1987': ['fp'],
+      '1988': ['fp'],
+      '1989': ['fp']
+    },
+    'Omni Turbo': {
```
Comment (Bjorn248) [inline common.js:4768]: This is not a Chrysler

File: common.js [hunk 40]
```diff
@@ -4432,6 +4431,361 @@ const allSoloCars = {
...
+      '1987': ['fp'],
+      '1988': ['fp'],
+      '1989': ['fp'],
+      '1990': ['fp'],
+      '1991': ['fp'],
+      '1992': ['fp'],
+      '1993': ['fp'],
+      '1994': ['fp']
+    },
+    'Talon & Laser (Turbo, FWD & AWD)': {
```
Comment (Bjorn248) [inline common.js:4781]: Talon is not a Chrysler

**Review by Bjorn248 — CHANGES_REQUESTED** [review]
Need to remove non-Chrysler models

**Comment by Bjorn248** [comment]
Thanks!

**Comment by hasalassh** [comment]
@Bjorn248 Thank You...!

**Pull request merged**